doc/dev/osd_internals/erasure_coding/developer_notes.rst

```diff
@@ -207,9 +207,9 @@ set in the erasure code profile, before the pool was created.
   ceph osd erasure-code-profile set myprofile \
      directory=<dir>         \ # mandatory
      plugin=jerasure         \ # mandatory
-     m=10                    \ # optional and plugin dependant
-     k=3                     \ # optional and plugin dependant
-     technique=reed_sol_van  \ # optional and plugin dependant
+     m=10                    \ # optional and plugin dependent
+     k=3                     \ # optional and plugin dependent
+     technique=reed_sol_van  \ # optional and plugin dependent
```

Comment:
i think "dependant" is not a typo. it's just less commonly used than "dependent". see https://www.merriam-webster.com/dictionary/dependant and https://www.lexico.com/en/definition/dependant

@johnwilkins what do you think?

Reply:
@tchaikov fair enough, I'm not a native speaker of english, just let me know which version you'd prefer

Reply:
neither am i =( . let's wait for more inputs before moving forward.

Reply:
I'd not seen "dependant" before, https://www.merriam-webster.com/dictionary/dependant#note-1 suggests it used to be used in particular cases of British English